src/game/bondmove.c

```diff
@@ -1946,10 +2022,34 @@ void bmoveProcessInput(bool allowc1x, bool allowc1y, bool allowc1buttons, bool i
 
 	// Look ahead
 	if (g_Vars.currentplayer->pausemode == PAUSEMODE_UNPAUSED) {
-		lookahead = -4;
+			lookahead = -4;
+			offbike = g_Vars.currentplayer->bondmovemode == MOVEMODE_WALK
+					|| g_Vars.currentplayer->bondmovemode == MOVEMODE_GRAB;
 
-		offbike = g_Vars.currentplayer->bondmovemode == MOVEMODE_WALK
-			|| g_Vars.currentplayer->bondmovemode == MOVEMODE_GRAB;
+#ifndef PLATFORM_N64
+			// Camera recenter vertical binding
+			static bool prev_reset_pressed = false;
+			bool reset_pressed = inputResetCameraPressed(cidx);
+
+			if (reset_pressed && !prev_reset_pressed) {
+				if (g_Vars.currentplayer) {
+					if (g_Vars.currentplayer->insightaimmode) {
+						// Recenter crosshair if in crosshair aim mode
+						g_Vars.currentplayer->swivelpos[0] = 0.0f;
+						g_Vars.currentplayer->swivelpos[1] = 0.0f;
+					}
+
+					if (offbike && !g_Vars.currentplayer->insightaimmode) {
+						g_Vars.currentplayer->cachedlookahead = bmoveCalculateLookahead();
+						g_Vars.currentplayer->docentreupdown = true;
+						g_Vars.currentplayer->automovecentre = false;
+						g_Vars.currentplayer->speedverta = (g_Vars.currentplayer->vv_verta < g_Vars.currentplayer->cachedlookahead) ? 7.0f : -7.0f;
+						s_recenterfast = 1;
+					}
+				}
+			}
+			prev_reset_pressed = reset_pressed;
```

Comment:
Maybe whitespace? Also, don't use `snake_case` in the game code, unless it's for a prefix like `g_`.